### PR TITLE
[IntelliJ Plugin] Add plugin grammar changes

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
This PR adds plugin support for latest string function invocation literal and also adds a minor bug fix.

### CODE REVIEW

File: tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaStringFunctionInvocationReference.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Shan1024) [inline tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaStringFunctionInvocationReference.java]: Update the year to 2019.

Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/gen/io/ballerina/plugins/idea/psi/BallerinaStringFunctionInvocationReference.java]: Updated.

File: tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/grammar/Ballerina.bnf [hunk 2]
```diff
@@ -390,7 +390,7 @@ Statement ::=
...
 // NULL_LITERAL is added for match pattern -  null => return "error";
-private StatementRecover ::= !(BOOLEAN_LITERAL|QUOTED_STRING_LITERAL|SYMBOLIC_STRING_LITERAL|DECIMAL_INTEGER_LITERAL|HEX_INTEGER_LITERAL|OCTAL_INTEGER_LITERAL|BINARY_INTEGER_LITERAL|NULL_LITERAL|int|string|float|decimal|boolean|byte|any|anydata|json|xml|xmlns|map|table|function|stream|'('|'{'|'}'|';'|typedesc|future|var|while|match|foreach|continue|break|fork|try|throw|return|abort|aborted|committed|retry|fail|lock|transaction|if|forever|object|check|error|panic|from|worker|record|identifier|wait)
+private StatementRecover ::= !(BOOLEAN_LITERAL|QUOTED_STRING_LITERAL|SYMBOLIC_STRING_LITERAL|DECIMAL_INTEGER_LITERAL|HEX_INTEGER_LITERAL|OCTAL_INTEGER_LITERAL|BINARY_INTEGER_LITERAL|NULL_LITERAL|DECIMAL_FLOATING_POINT_NUMBER|HEXADECIMAL_FLOATING_POINT_LITERAL|int|string|float|decimal|boolean|byte|any|anydata|json|xml|xmlns|map|table|function|stream|'('|'{'|'}'|';'|typedesc|future|var|while|match|foreach|continue|break|fork|try|throw|return|abort|aborted|committed|retry|fail|lock|transaction|if|forever|object|check|error|panic|from|worker|record|identifier|wait)
```
Comment (Shan1024) [inline tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/grammar/Ballerina.bnf]: `OCTAL_INTEGER_LITERAL` has been removed from the language. Please send another PR updating those changes.

Comment (NipunaRanasinghe) [reply tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/grammar/Ballerina.bnf]: will do
